java/core/src/test/java/sleeper/core/partition/PartitionFactoryTest.java

```diff
@@ -139,9 +139,9 @@ void shouldSpecifyParentThenChildPartitionsWithTwoDimensions() {
                         .build(),
                 Partition.builder()
                         .rowKeyTypes(rowKeyTypes)
-                        .region(Region.from(List.of(
-                                rangeFactory.createRange(key1, "aaa", null),
-                                rangeFactory.createRange(key2, "bbb", null))))
+                        .region(new Region(List.of(
+                        rangeFactory.createRange(key1, "aaa", null),
+                        rangeFactory.createRange(key2, "bbb", null))))
```

Comment:
The indentation here looks a bit strange. I guess checkstyle isnt picking it up since this is a test class?

Reply:
Fixed, thanks